### ISSUE TITLE
Accessibility, Semantics and Screen Reader

### ISSUE THREAD

**Issue by mariapapag** [flutter/flutter#107956]
## Use case

Example: User navigates to a new page. A loading page appears before the new page does. The loading page semantics are read out by the screen reader but the audio is cut off when navigation occurs, so user does not fully hear "loading", they hear "lo-" which is confusing. 

## Proposal

Is there a way to force screen reader to complete speaking out the semantics hint or label before navigating to another screen?

**Comment by timsneath** [comment]
Thanks for the question, @mariapapag. Are you talking about web, desktop or mobile? Which OS specifically?

**Comment by mariapapag** [comment]
Mobile, for iOS and Android

**Comment by exaby73** [comment]
Hello @mariapapag. Thanks for filing this issue. Can you provide a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) highlighting this? Can you also confirm, if what you proposed is native behavior? For example: does the screen reader read the full word "loading" on a native app or is this also something you need to especially configure. Sorry for my lack of knowledge around this topic
